packages/executor/src/modules/eth.ts

```diff
@@ -480,7 +480,7 @@ export class Eth {
         let fromBlockNumber = blockNumber - this.config.receiptLookupRange;
         // underflow check
         if (fromBlockNumber < 0) {
-          fromBlockNumber = blockNumber;
+          fromBlockNumber = 0;
```

Comment:
Wont this cause delays/timeout?

Reply:
no, this is for the cases when the blockchain is new and `receiptLookupRange` is more than the number of blocks in blockchain (happens in testing mode only usually)